src/app/locales/pt_BR.ts

```diff
@@ -0,0 +1,538 @@
+  },
+  TrackByFn: {
+    action: 'Use TrackByFunction em vez de TrackByFn',
+  },
+  'i18n Pipe Change': {
+    action:
+      'Se você usar os pipes `date, `currency, `decimal ou `percentage`, na v5 você verá algumas pequenas mudanças no formato. Para aplicativos que usam `locales diferentes de en-us, você precisará importá-lo e, opcionalmente, `locale_extended_fr` de `@angular/common/i18n_data/locale_fr` e registerLocaleData(local).',
+  },
+  gendir: {
+    action:
+      'Não confie no uso de `gendir`, em vez disso use` skipTemplateCodeGen`. <a href=https://github.com/angular/angular/issues/19339#issuecomment-332607471" target="_blank"> Leia mais </a>',
+  },
+  'Dynamic ngUpgrade': {
+    action:
+      'Substitua `downgradeComponent`,` downgradeInjectable`, `UpgradeComponent` e` UpgradeModule` importados de `@ angular/upgrade`. Use a nova versão em `@angular/upgrade/static`',
+  },
+  'Animations in Core': {
+    action:
+      'Se você usar serviços para animações ou ferramentas de @angular/core, você deve importá-los de @angular/animations',
+  },
+  ngOutletContext: {
+    action: 'Substitua `ngOutletContext` por` ngTemplateOutletContext`.',
+  },
+  collectionChangeRecord: {
+    action: 'Substitua `CollectionChangeRecord` por` IterableChangeRecord`',
+  },
+  Renderer: {
+    action: 'Em todos os lugares que você estiver usando o Renderer, substitua-o por Renderer2',
+  },
+  'Router Query Params': {
+    action: 'Se você utilizar `preserveQueryParams`, substitua por ´queryParamsHandling` .',
+  },
+  Http: {
+    action:
+      'Se você usar o antigo módulo `HttpModule` e o serviço` Http`, substitua-o pelo serviço` HttpClientModule` e `HttpClient`. HttpClient melhora a ergonomia por padrão (você não precisa mais usar um `map` para converter a resposta para JSON) e agora suporta tipos de retorno e interceptores. Você pode ler mais em [angular.io] (https://angular.io/guide/http).',
+  },
+  'DOCUMENT in @angular/platform-browser': {
+    action: 'Se você usar DOCUMENT de `@angular/platform-browser`, deve começar a importá-lo de `@angular/common`',
+  },
+  ReflectiveInjector: {
+    action: 'Substitua o `ReflectiveInjector` por `StaticInjector`',
+  },
+  Whitespace: {
+    action:
+      'Aproveite o uso de `preserveWhitespaces` com o valor de` off` em` tsconfig.json` na propriedade` angularCompilerOptions`. Na versão 6, ele é inicializado por padrão com `off`.',
+  },
+  'node 8': {
+    action: 'Certifique-se de usar o <a href="https://nodejs.org/en/" target="_blank"> Node 8 ou posterior. </a>',
+  },
+  'Update to CLI v6': {
+    action:
+      'Atualize a versão local do Angular CLI e migre a configuração para <a href="https://github.com/angular/angular-cli/wiki/angular-workspace" target="_blank"> para o novo formato angular. json </a>, executando o seguinte comando: <br/> <br/> `${packageManagerInstall}@angular/cli@6`<br/>` ng update @angular/cli@6` <br/>',
```

Comment:
just a question: Is ` between cli@6 and <br/> expected?

Reply:
Done!